docs/DeveloperGuide.md

```diff
@@ -29,7 +29,7 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 ### Architecture
 
-<img src="images/ArchitectureDiagram.png" width="280" />
+![ArchitectureDiagram.png](images/ArchitectureDiagram.png)
```

Comment:
any reason for these changes? you want to make the images bigger for easier readability?

Reply:
oh didn't realise it made it bigger, just wanted to standardise it/ make the preview available in IntelliJ lol. I can revert the changes